docs/src/libraries/osmosis-gamm-lper.md

```diff
@@ -0,0 +1,61 @@
+# Osmosis GAMM LPer library
+
+The **Valence Osmosis GAMM LPer library** library allows to **join a pool** on **Osmosis**, using the **GAAM module** (Generalized Automated Market Maker), from an **input account**, and transfer the **position** to an **output account**.
```

Comment:
to avoid confusion I wouldn't use the term position here, as that's particular to CL. I would say transfer the LP tokens or GAMM tokens